### PR TITLE
suggesters: add weights for experiment suggester

### PR DESCRIPTION
**Pull request by harunurhan**
* INSPIR-2406

### CODE REVIEW

**Comment by michamos** [comment]
I don't think this is the right approach, here you're boosting the legacy name because you assume that's what people know, but it's very often not the case.

**Comment by harunurhan** [comment]
@michamos complains came when they were trying to auto-complete with legacy name AFAIK, that's why we decided to try this out temporarily while we we are decide what to do with suggesters in general (only prefix based suggestion problem)

**Comment by michamos** [comment]
OK I see :+1: but this is a symptom of a much more general problem

File: backend/tests/unit/records/marshmallow/experiments/test_es.py [hunk 1]
```diff
@@ -45,44 +45,19 @@ def test_experiment_serializer_populates_experiment_suggest():
...
+        {"input": "ACC", "weight": 1},
+        {"input": "COLLABORATION", "weight": 1},
+        {"input": "EXP SHORT NAME", "weight": 1},
+        {"input": "Experiment value", "weight": 1},
+        {"input": "INST_VALUE", "weight": 1},
+        {"input": "{Long Name}", "weight": 1},
+        {"input": "NAME_V1", "weight": 1},
+        {"input": "NAME_V2", "weight": 1},
+        {"input": "NAME_V3", "weight": 1},
+        {"input": "LEGACY-NAME", "weight": 5},
```
Comment (drjova) [inline backend/tests/unit/records/marshmallow/experiments/test_es.py]: don't we need to change something on UI part?

Comment (harunurhan) [reply backend/tests/unit/records/marshmallow/experiments/test_es.py]: as far as I can see from the docs, input value that matched always comes in `options[].text`, weight is just used for matching.

**Pull request merged**